core/shared/utils/bh_leb128.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
this file seems mostly copy-and-paste from wasm_loader.c.
please retain the original copyright notice.

Reply:
good point, updated.